packages/terra-cell-grid/src/CellGrid.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './CellGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The children passed to the component
```

Comment:
Should these children be cells? Do we care?

Reply:
Updated.